src/api/spec/models/obs_factory/distribution_strategy_factory_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ObsFactory::DistributionStrategyFactory do
+  let(:project) { create(:project) }
+
+  describe '::new' do
+    context 'called with no parameters' do
+      subject { ObsFactory::DistributionStrategyFactory.new }
+
+      it { is_expected.to be_kind_of(ObsFactory::DistributionStrategyFactory) }
+    end
+
+    context 'called with project parameter' do
+      subject { ObsFactory::DistributionStrategyFactory.new(project: project) }
+
+      it { is_expected.to be_kind_of(ObsFactory::DistributionStrategyFactory) }
+      it { is_expected.to have_attributes(project: project) }
```

Comment:
Same here, these are stdlib things...

Reply:
What you mean is not Ruby stdlib, but Rails's [ActiveModel::Model's new](http://api.rubyonrails.org/classes/ActiveModel/Model.html#method-c-new). For Ruby classes you need to define an initializer to add additional behaviour, like assigning a project if you want that.

Thinking about it we could also include ActiveModel::Model to get this behaviour. :thinking: